scala-core-6/src/test/scala/com/baeldung/scala/maps/MapsUnitTest.scala

```diff
@@ -24,11 +24,6 @@ class MapsUnitTest extends WordSpec with Matchers {
       val map: Map[Int, String] = immutable.Map[Int, String]()
       map shouldBe empty
 
-    }
-    "create empty map with apply" in {
```

Comment:
moved this to src/test/scala directory but afterward had to remove this as this doesn't compile.